blockchain/blockchain.go

```diff
@@ -0,0 +1,182 @@
+//go:generate abigen --abi ../resources/Agent.abi --pkg blockchain --type Agent --out agent.go
+
+package blockchain
+
+import (
+	"bytes"
+	"crypto/ecdsa"
+	"encoding/hex"
+	"encoding/json"
+	"github.com/coreos/bbolt"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/singnet/snet-daemon/config"
+	"github.com/singnet/snet-daemon/db"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+)
+
+const (
+	JobPendingState    = "PENDING"
```

Comment:
golint/govet should be complaining about exported fields not having docstring comments. we should follow community best practices and also run go vet / go lint as part of checkin.